cleanup_job.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+import logging
+import requests
+from datetime import datetime, timedelta
+
+from cachito.workers.requests import get_requests_session
+from cachito.workers.config import get_worker_config
+from cachito.errors import CachitoError
+
+log = logging.getLogger(__name__)
+
+auth_session = get_requests_session(auth=True)
+session = get_requests_session()
+config = get_worker_config()
+
+
+def main():
+    """
+    Get all completed requests from the Cachito API.
+
+    :raise CachitoError: if the request to the Cachito API fails
+    """
+    # Get all completed requests
+    response = session.get(config.cachito_api_url + 'requests?state=complete')
+    if not response.ok:
+        raise CachitoError('Could not reach cachito API to get all completed requests')
+    json_response = response.json()
+    identify_stale_requests(json_response)
+
+
+def identify_stale_requests(requests_json):
```

Comment:
This function name is misleading because the function actually modifies the requests, but you wouldn't expect that by the function name.